### ISSUE TITLE
Add Timestamps for Task

### ISSUE THREAD

**Issue by dennypradipta** [dennypradipta/mern-with-jwt#12]
Now that we have a fully functional task system that connects to the REST API, there is some minor changes to the data stored in MongoDB.

So I need fields in the Model that stores timestamps for saved task, such as created_at and modified_at.

But I am hesitating to decide whether to add one field called done_at or not. I mean sure, modified_at will store timestamps when there is modifications to the task such as edit and marked as done. But, what is the best practice? Should you separate done_at with modified_at or combine them in modified_at?

The possibility is that when you have done_at fields, you can gather data of "Task cleared before 31st October". If you only have modified_at, you can only gather data of "Task modified before 31st October".

In conclusion: Add created_at and modified_at field in Todo Model, and decide whether to add one field called done_at or not.

What are your thoughts? Comment down below and maybe try some PR.

Thank you very much

**Comment by MihaiDruta** [comment]
I think that `created_at` and `updated_at` should be enough. You could add a field smthng like `finished` of boolean type, and checking it whenever displaying some info (ex. if `finished` is `true` print "Task finished at (the value of `updated_at`)", or if `finished` is `false` display "Task updated at ..."). 

And I think these values (`created_at` and `updated_at`) should change automatically using some sort of lifecycle callbacks.

**Comment by MihaiDruta** [comment]
Yeah, I think I could take it. I'm in :)
